app/shared/api-services/poll/answer-factory.js

```diff
@@ -13,11 +13,7 @@ angular.module('apiServices.answer.factory', [
       return $http.delete(answerUrl + deletedAnswer.id);
     });
     $q.all(deletePromise).then(function(response) {
-      if(response.status === 201) {
-        return response;
-      } else {
-        return $q.reject('Delete unsucessful');
-      }
+      return response;
```

Comment:
Did we remove everywhere that we're checking for error responses or is that logic somewhere else?